packages/nouns-contracts/contracts/client-incentives/Rewards.sol

```diff
@@ -354,26 +357,17 @@ contract Rewards is
             uint endBlock = max(proposals[i].endBlock, proposals[i].objectionPeriodEndBlock);
             require(block.number > endBlock, 'all proposals must be done with voting');
 
-            // skip non eligible proposals
-            if (proposals[i].forVotes < (proposals[i].totalSupply * proposalEligibilityQuorumBps_) / 10_000) {
-                delete proposals[i];
-                continue;
-            }
-
-            // proposal is eligible for reward
-            ++t.numEligibleProposals;
-
             uint256 votesInProposal = proposals[i].forVotes + proposals[i].againstVotes + proposals[i].abstainVotes;
             t.numEligibleVotes += votesInProposal;
         }
 
+        t.numEligibleProposals = proposals.length;
```

Comment:
nit: can we move this up, right after we get the proposals list?

Reply:
69bc31a158cc629dccdf27c54acf6ed69b4b5d22